src/ansys/dpf/post/harmonic_mechanical_simulation.py

```diff
@@ -125,6 +125,8 @@ def _get_result_workflow(
             equivalent_op = self._model.operator(name="eqv_fc")
             wf.add_operator(operator=equivalent_op)
             # If a strain result, change the location now
+            # TBD: Why do we put the the equivalent operator
+            # before the averaging operator for strain results?
```

Comment:
Hey @janvonrickenbach,
If I remember correctly, it should be because this is what is done in the Mechanical workflow.

Reply:
Thanks, this is also what Camille mentioned. I will add a note.